python/docs/source/reference/pyspark.sql/io.rst

```diff
@@ -52,3 +52,14 @@ Input/Output
     DataFrameWriter.saveAsTable
     DataFrameWriter.sortBy
     DataFrameWriter.text
+    DataFrameWriterV2.using
+    DataFrameWriterV2.option
+    DataFrameWriterV2.options
+    DataFrameWriterV2.tableProperty
+    DataFrameWriterV2.partitionedBy
+    DataFrameWriterV2.create
+    DataFrameWriterV2.replace
+    DataFrameWriterV2.createOrReplace
+    DataFrameWriterV2.append
+    DataFrameWriterV2.overwrite
+    DataFrameWriterV2.overwritePartitions
```

Comment:
Ah, actually these cannot be imported because it's missing in `pyspark/sql/__init__.py`. Let me revert it for now since the CI is broken, and technically the tests didn't pass here.

Sorry it was my bad ...

Reply:
sorry, I should make it a WIP draft